OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -593,6 +598,11 @@ public boolean isCourseVideosEnabled() {
         return getBoolean(COURSE_VIDEOS_ENABLED, true);
     }
 
+    /**
+     * On the time when deprecated activities {@link org.edx.mobile.view.MyCoursesListActivity},
+     * {@link org.edx.mobile.view.CourseDashboardActivity} and {@link org.edx.mobile.view.CourseOutlineActivity}
+     * will be removed, this function will no longer be of any use.
+     */
     public boolean isTabsLayoutEnabled() {
```

Comment:
`@Deprecated` can be added the top of this line.

Reply:
`At the time` instead of `On the time`.

Reply:
I will not prefer to add `@Deprecated` because there is no alternate available of it and developer suppose to use this flag in v2.13. Strike through on it will not make any sense.

Reply:
ok, fair enough.